src/main/java/utils/elements/ElementUtils.java

```diff
@@ -10,4 +13,30 @@ public static PointOption getCenterElementPointOption(WebElement element) {
         int x = element.getRect().getWidth()/2;
         return new PointOption().point(element.getLocation().moveBy(x, y));
     }
+
+    public static void clickOnElement(WebElement element) {
```

Comment:
Rename this method in order to make more clear what happening in this method (e.g. clickUsingAction(Webelement element) {} )